### PR TITLE
Update swagger doc

### PR DESCRIPTION
**Pull request by PolinaStamenova**
fix #issuenumber

# Update swagger documentation

- Add some missing properties into the schema.
- Remove extra added properties from the schema.
- Rename some descriptions into the paths.

#### Only select the appropriate.

- [ ] **Model testing.**
- [ ] **Request testing and swagger doc update.**
- [ ] **System testing.**
- [ ] **No tests required.**

### CODE REVIEW

File: backend/swagger/v1/swagger.yaml [hunk 1]
```diff
@@ -324,20 +331,20 @@ paths:
...
+          description: Successful operation - Survey with a given ID returned.
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Survey'
   '/api/v1/answers_surveys':
     post:
-      summary: Create Answers Survey
-      description: Create Answers Survey
+      summary: Add a new Answers Survey to the store
```
Comment (edimossilva) [inline backend/swagger/v1/swagger.yaml:341]: This one I would return to `Create Answers Survey`.
Reasons
- The term `Add` usually is related to `array` or `list` data structures
- The term `store` usually is related to a storage pattern in front ent.

File: backend/swagger/v1/swagger.yaml [hunk 2]
```diff
@@ -349,7 +356,7 @@ paths:
               $ref: '#/components/schemas/CreateAnswersSurveyRequestBody'
   '/api/v1/answers_surveys/{id}':
     get:
-      summary: Get Answers Survey
+      summary: List a single Answers Survey by ID
```
Comment (edimossilva) [inline backend/swagger/v1/swagger.yaml:359]: List word is associated to arrays, so I would go with `Get Answers Survey by ID` or `Retrieve Answers Survey by ID`

File: backend/swagger/v1/swagger.yaml [hunk 3]
```diff
@@ -361,21 +368,21 @@ paths:
...
+          description: Successful operation - AnswersSurvey returned.
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/AnswersSurveyWithAnswers'
 
   '/api/v1/answers':
     post:
-      summary: Create Answers
+      summary: Add a new Answer to the store
```
Comment (edimossilva) [inline backend/swagger/v1/swagger.yaml:379]: same here

**Review by edimossilva — CHANGES_REQUESTED** [review]
Nice, thanks for working on this.
I have some small suggestions

**Comment by iurynogueira** [comment]
Issue number pls.

**Review by edimossilva — APPROVED** [review]

**Pull request merged**